go/test/endtoend/vreplication/cluster_test.go

```diff
@@ -252,8 +252,8 @@ func downloadDBTypeVersion(dbType string, majorVersion string, path string) erro
 		versionFile = "mysql-8.0.28-linux-glibc2.17-x86_64-minimal.tar.xz"
 		url = "https://dev.mysql.com/get/Downloads/MySQL-8.0/" + versionFile
 	} else if dbType == "mariadb" && majorVersion == "10.10" {
-		versionFile = "mariadb-10.10.2-linux-systemd-x86_64.tar.gz"
-		url = "https://mirror.rackspace.com/mariadb/mariadb-10.10.2/bintar-linux-systemd-x86_64/" + versionFile
+		versionFile = "mariadb-10.6.12-linux-systemd-x86_64.tar.gz"
```

Comment:
```suggestion
		versionFile = "mariadb-10.10.3-linux-systemd-x86_64.tar.gz"
```

Reply:
@arthurschreiber We're deliberate here to use the latest LTS so it's not EOL as quickly as `10.10` will be (only 1 year support for that one).

Reply:
Hey @arthurschreiber, we want to use `10.6.12` as it offers a long term support of 5 years, as opposed to 1 year for `10.10.3`

Reply:
Ah, ok. It's just confusing because the line before explicitly checks for `10.10`, but then we end up using `10.6` here.

Reply:
@arthurschreiber ah, that's a good catch, I think we should update that check then.

Reply:
Didn't realize here that the version is specified somewhere else actually used in the test and not here:

```
go/test/endtoend/vreplication/vreplication_test.go:     sourceKsOpts["DBTypeVersion"] = "mariadb-10.10"
```

Reply:
Good catch, i'll fix that!

Reply:
https://github.com/vitessio/vitess/blob/c7605a147fa548c7bf1c0623a23688fd41b1ded5/go/test/endtoend/vreplication/vreplication_test.go#L350 Is the only place where this is coming from, I believe.